### ISSUE TITLE
No access to ResNet50 caffe model

### ISSUE THREAD

**Issue by jonahduncan** [motokimura/cowc_car_counting#4]
Thank you for this repository @motokimura! 

I'm currently trying to recreate the results as per your instructions. I want to use the ResNet50 pre-trained model, but the document provided through the link you gave no longer exists. The only alternative links I've found have been through Baidu, which I'm struggling to access. Do you happen to have a copy of the caffemodel? I'd prefer to avoid training from scratch. 

If you don't, feel free to close this issue as it's not an issue with your repository.

**Comment by motokimura** [comment]
Sorry for my late response.

Unfortunately, I removed the trained weight accidentally..
I have never tried, but I guess the accuracy is not so bad even without imagenet pre-training.

Or, you can try other deep learning frameworks than Chainer.
E.g., PyTorch is easier to use pre-trained model.
You can still use scripts and notebooks of this repository for data preparation, evaluation, and visualization (training is much simpler than these pre- and post-processes).

Hope this helps. Thank you for your interest!

**Comment by jonahduncan** [comment]
Thanks for the recommendation. I appreciate you getting back to me (regardless of timing!)
